pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -234,6 +234,14 @@ protected void validateNamespaceName(String property, String namespace) {
         }
     }
 
+    protected void validateGlobalNamespaceOwnership(String property, String namespace) {
+        try {
+            this.namespaceName = NamespaceName.get(property, namespace);
+            validateGlobalNamespaceOwnership(this.namespaceName);
+        } catch (RestException re) {
```

Comment:
@foreversunyao : it seems that we almost do nothing here than just re-throwing the exception. so do we really need this try-catch block here?

Reply:
yeah, you are right, will fix it

Reply:
@sijie , after I checked the function **validateGlobalNamespaceOwnership**, I think we could keep the catch thing and also need one more catch to get the UNAVALIBALE Exceptions ?

Reply:
One of the reason we handle the RestException separately is that we want to catch before the `catch(Exception)` block which will apply the generic error code instead of the more accurate error.